src/ServicePulse.Host/vue/src/composables/serviceServiceControlUrls.ts

```diff
@@ -72,7 +79,7 @@ export function useServiceControlUrls() {
     window.localStorage.setItem("scu", serviceControlUrl.value);
     console.debug(`ServiceControl Url found in QS and stored in local storage: ${serviceControlUrl.value}`);
   } else if (window.localStorage.getItem("scu")) {
-    serviceControlUrl.value = window.localStorage.getItem("scu");
+    serviceControlUrl.value = window.localStorage.getItem("scu") || undefined;
```

Comment:
|| undefined doesn't make sense to me...? If the getItem doesn't find the key, won't it return undefined anyway?